Assets/Events/Stories.json

```diff
@@ -770,4 +776,454 @@
+                    "article": "Yet again, our adventurers have returned with a new map and not in hand. \"Not much further now! What lies at the end of this trail is worth.\" I swear, if this one doesn't lead to something cool, I'm quitting.",
+                    "image": "map",
+                    "outcomes": [
+                      {
+                        "type": "QuestAdded",
+                        "quest": {
+                          "name": "cartographer-quest-3",
+                          "title": "Follow THIS Map, I Guess",
+                          "description": "This is the one, I can feel it...",
+                          "image": "map",
+                          "location": "Dock",
+                          "adventurers": 2,
+                          "baseTurns": 3,
+                          "wealthMultiplier": 0.5,
+                          "reward": "It's gotta lead to something!",
+                          "completedEvent": {
+                            "name": "cartographer-4",
+                            "headline": "\"This Is Just Too Many Maps!\"",
+                            "article": "Our adventurers have returned, this time not with a single map, but hundreds! At the location specified, there was a huge stash of maps, each with the note \"Nah, really, THIS one is the real one. You gotta believe me, untold riches, magic powers, that kinda stuff.\" Frankly, we've had enough, it wasn't even a 'the real treasure was the friends we made' deal! Now need to work out what to do with all these maps.",
+                            "image": "map",
+                            "outcomes": [
+                              {
+                                "type": "SetFlag",
+                                "flag": "StoryActive",
+                                "value": false,
+                                "customDescription": "Maybe if we just sell these maps off, some other adventurers can make use of them."
+                              },
+                              {
+                                "type": "CardUnlocked",
+                                "blueprint": "Cartographer"
+                              }
+                            ]
+                          }
+                        }
+                      }
+                    ]
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    ]
+  },
+  {
+    "name": "water-purity-1",
+    "headline": "Wastage in Well causes Wicked Water",
+    "article": "An batch of expired potions has been illegally dumped into our well, resulting in tainted water! We need to search for a solution, quick!\n<b>Diviners: </b>Our scriptures tell of a sacred site, the 'Fountain of Purity', a single drop from it may solve our problems.\n<b>Performers: </b>Orrrr, we could 'borrow' some of the brewing supplies from the orcs. It's a win win! We get to purified water, and everyone gets a little buzzed!",
```

Comment:
```suggestion
    "headline": "Well Wastage Causes Wicked Water",
    "article": "A batch of expired potions has been illegally dumped into our well, resulting in tainted water! We need to search for a solution, quick!\n<b>Diviners: </b>Our scriptures tell of a sacred site, the 'Fountain of Purity', a single drop from it may solve our problems.\n<b>Performers: </b>Orrrr, we could 'borrow' some brewing supplies from the orcs. It's a win win! We get to purified water, and everyone gets a little buzzed!",
```